lib/product-import.coffee

```diff
@@ -164,16 +164,14 @@ class ProductImport
         @_getWhereQueryLimit()
       )
       Promise.map(skuChunks, (skus) =>
-        new Promise (resolve, reject) =>
-          predicate = @_createProductFetchBySkuQueryPredicate(skus)
-          @client.productProjections
-          .where(predicate)
-          .staged(true)
-          .all()
-          .fetch()
-          .then ({ body: { results } }) ->
-            resolve(results)
-          .catch (err) -> reject(err)
+        predicate = @_createProductFetchBySkuQueryPredicate(skus)
+        @client.productProjections
+        .where(predicate)
+        .staged(true)
+        .all()
```

Comment:
Hello @butenkor - do I understand it correctly that `.all()` fetches [all results](https://github.com/sphereio/sphere-node-sdk/blob/master/src/coffee/services/base.coffee#L227) matching predicate and with this `.perPage(100)` you want to increase a page size so we will reduce the number of API calls?